tests/fft_test.py

```diff
@@ -353,5 +353,51 @@ def testRfftfreqErrors(self, n):
       lambda: func(n=10, d=n)
     )
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+    {"testcase_name": "_shape={}".format(
+      jtu.format_shape_dtype_string(shape, dtype)),
+      "dtype": dtype, "shape": shape, "rng_factory": rng_factory}
+    for rng_factory in [jtu.rand_default]
+    for dtype in all_dtypes
+    for shape in [[9], [10], [101], [102], [3, 5], [3, 17], [5, 7, 11]]))
+  def testFftshift(self, shape, dtype, rng_factory):
+    rng = rng_factory()
+    args_maker = lambda: (onp.array(onp.random.randn(*shape), dtype=dtype),)
+    np_op = np.fft.fftshift
+    onp_op = onp.fft.fftshift
+    np_fn = lambda arg: np_op(arg)
+    onp_fn = lambda arg: onp_op(arg)
+    # Numpy promotes to complex128 aggressively.
+    self._CheckAgainstNumpy(onp_fn, np_fn, args_maker, check_dtypes=False,
+                            tol=1e-4)
+    self._CompileAndCheck(np_fn, args_maker, check_dtypes=True)
+    # Test gradient for differentiable types.
+    if dtype in inexact_dtypes:
+      tol = 0.15  # TODO(skye): can we be more precise?
+      jtu.check_grads(np_fn, args_maker(), order=2, atol=tol, rtol=tol)
```

Comment:
I think this can all be replaced by just:
```python
    self._CheckAgainstNumpy(onp_fn, np_fn, args_maker, check_dtypes=True)
```

The comment on complex128 is not relevant to fftshift/iffshift. And we not really need to check compiling or gradients here, since these functions are built-up out of the independently tested `jax.numpy.roll`.

Reply:
Nice comment, updated according to this (although there might be more culprits with the same problem laying around in this file, that's for later)